### PR TITLE
BIZ-551 Link from the email leads to the "http" instead of "https"

### PR DESCRIPTION
**Pull request by dmitrij-borchuk**

### CODE REVIEW

File: backend/services/emailTemplates/conversation/text.ejs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Video from <%= authorName %>:
+<%= title %>
+<%= message %>
+Follow the link to see the video: <%= buttonLink %>
+BizKonect - Video Messaging Platform for Technology Sales. Record, send & track one-to-one personalized video messages direct to your Customer’s Email, Social Media, or Text.
```
Comment (aleksspeaker) [inline backend/services/emailTemplates/conversation/text.ejs]: Is it ok to start new line with '+' or it's better to place it on the line before?

Comment (aleksspeaker) [inline backend/services/emailTemplates/conversation/text.ejs:5]: As I see, it's a new file.
Were we missing this email template?

**Review by aleksspeaker — APPROVED** [review]

File: backend/services/emailTemplates/conversation/text.ejs (hunk 1 above)
Comment (dborchuk) [reply backend/services/emailTemplates/conversation/text.ejs:5]: Yes

Comment (aleksspeaker) [reply backend/services/emailTemplates/conversation/text.ejs:5]: Testing slack...

**Pull request merged**